prow_config.yaml

```diff
@@ -24,3 +24,25 @@ workflows:
     - test/*
     params:
       registry: "gcr.io/kubeflow-images-public"
+  - app_dir: kubeflow/katib/test/workflows
+    component: workflows-v1alpha2
+    name: e2e-v1alpha2
+    job_types:
+      - presubmit
+    include_dirs:
+    - pkg/*
```

Comment:
Can we configure the conditions such that only v1alpha1 tests are run if v1alpha1 code is changed, and similarly for v1alpha2?

Reply:
that will list two many files, especially we plan to share common files between the two versions, such as `Dockerfile`

Reply:
it is a good idea to trigger corresponding version workflow when related code changed. maybe we can implement it when we refactor katib dir.

Reply:
#472 trace it